modules/serverless-logic-openAPI-function-definition.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
```

Comment:
List the assembly metadata

// * serverless-logic/serverless-logic-managing-services/serverless-logic-configuring-openapi-services